azure-pipelines.yml

```diff
@@ -44,7 +44,30 @@ stages:
           - script: |
               npm run build --prod
             displayName: 'npm build'
-
+          
+          - script: |
+             npm install jest jest-preset-angular @types/jest --save-dev
+             npm install jest-junit --save-dev
+             npm test --reporters=jest-junit --reporters=default --coverage --coverageReporters=cobertura --coverageReporters=html
+            continueOnError: false
+            displayName: 'Unit test with Jest'
+            workingDirectory: 'src/app'
+ 
+          - task: PublishTestResults@2
+            condition: succeededOrFailed()
+            inputs:
+             testRunner: JUnit
+             testResultsFiles:  junit.xml
+            displayName: 'Publish Unit Test Results'
+          
+          - task: PublishCodeCoverageResults@1
+            inputs:
+              codeCoverageTool: 'Cobertura'
+              summaryFileLocation: '$(System.DefaultWorkingDirectory)/**/*coverage.xml'
+              reportDirectory: 'coverage/my-app/'
+              additionalCodeCoverageFiles: index.html
+            displayName: 'Code Coverage'
```

Comment:
```suggestion
            displayName: 'Unit Test Code Coverage'
```

Reply:
Done the change. 
In addition , removed **additonalCoverageFiles** from unit test step as it is not required.